db/init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE IF NOT EXISTS rha;
+
+CREATE TABLE events (
+    event_id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255),
+    description TEXT,
+    start_time DATETIME,
+    event_type INT,
+    event_location_id BIGINT,
+    min_robins TINYINT,
+    max_robins TINYINT,
+    created_by VARCHAR(255),
+    updated_by VARCHAR(255),
```

Comment:
There can be multiple updates, we can keep the change log in a different table. We can also skip this altogether for now and handle this in future